doc/man7/EVP_PKEY-RSA.pod

```diff
@@ -110,7 +110,7 @@ When generating RSA keys, the following key generation parameters may be used.
 
 =over 4
 
-=item "bits" (B<OSSL_PKEY_PARAM_RSA_BITS>) <unsigned integer>
+=item "bits" (B<OSSL_PKEY_PARAM_RSA_BITS>) <size_t>
```

Comment:
Why?

This maps to OSSL_PKEY_PARAM_BITS which is an integer..
OSSL_PKEY_PARAM_MAX_SIZE & OSSL_PKEY_PARAM_SECURITY_BITS are also integers, so it seems strange to change this one..

Reply:
Also the integer parameters are not really sized. It is `unsigned integer` and not `unsigned int` for a reason.

Reply:
`unsigned integer` is correct here.

Reply:
There were inconsistencies w.r.t. the type of `OSSL_PKEY_PARAM_RSA_BITS`, which I resolved in favor of `size_t`.
See e.g., `providers/implementations/keymgmt/rsa_kmgmt.c`

Reply:
I'm not too surprised there are inconsistencies :(

Reply:
The integer parameters themselves do not have a size. The get/set functions automatically handle size conversions.

Reply:
The documentation doesn't specify C types, it specifies OSSL_PARAM types.  The type the C code uses is separate.

There are two `size_t` mistakes in the documentation which I'll fix.

Reply:
@paulidale  Already submitted #15178

Reply:
> There are two `size_t` mistakes in the documentation which I'll fix.

I only see one size_t in parameter list though.

Reply:
So reverted to `unsigned integer`.